PendingReleaseNotes

```diff
@@ -9,3 +9,15 @@
   New commands "pg cancel-force-recovery" and "pg cancel-force-backfill"
   restore default recovery/backfill priority of previously forced pgs.
 
+* A first release of Ceph for FreeBSD is available which contains a full 
+  set of features, other than Bluestore. It will run everything needed to
+  build a storage cluster. For clients, all access methods are available,
+  albeit CephFS is only accesable thru a Fuse implementation. RBD images 
+  can be mounted on FreeBSD systems thru rbd-ggate
+
+  Ceph versions are released thru the regular FreeBSD ports and packages
+  system. The most current version is available as: net/ceph-devel. 
+  Once Luminous goes into official release is will be availble as
```

Comment:
Since this is the release notes for luminous, should you just state it will be available via net/ceph and that future development releases will be available via net/ceph-devel?